resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/resources/ResourceTest.java

```diff
@@ -200,6 +222,16 @@ public ResourceTest(String name) {
 		super(name);
 	}
 
+	@Before
+	public void before() throws Exception {
+		setUp();
+	}
+
+	@After
+	public void after() throws Exception {
+		tearDown();
+	}
+
```

Comment:
I'd propose to make these methods `final` to avoid that subclasses overwrite them with setup/teardown functionality although sticking to JUnit 3 (such that the methods will never be called and added functionality is actually dead code). Please also document the mixture of JUnit 3 and JUnit 4 at these methods to ease comprehension for why the methods are there.

Reply:
@HeikoKlare  done